### PR TITLE
vulnerabilities controller test with mypy checked

### PR DESCRIPTION
**Pull request by Yuhu-kth**

### CODE REVIEW

File: simulation-system/libs/csle-common/tests/test_vulnerabilities_controller.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+
+class TestVulnControllerSuite:
+    """
+    Test VulnerabilitiesController
+    """
+
+    @patch("csle_common.util.emulation_util.EmulationUtil.connect_admin")
+    @patch("csle_common.util.emulation_util.EmulationUtil.execute_ssh_cmd")
+    def test_create_users(self, mock_execute_ssh_cmd, mock_connect_admin) -> None:
```
Comment (Limmen) [inline simulation-system/libs/csle-common/tests/test_vulnerabilities_controller.py:14]: This test should probably be renamed

**Review by Limmen — CHANGES_REQUESTED** [review]

File: simulation-system/libs/csle-common/tests/test_vulnerabilities_controller.py (hunk 1 above)
Comment (Yuhu-kth) [reply simulation-system/libs/csle-common/tests/test_vulnerabilities_controller.py:14]: oh right !

Comment (Yuhu-kth) [reply simulation-system/libs/csle-common/tests/test_vulnerabilities_controller.py]: fiexed

Comment (Limmen) [reply simulation-system/libs/csle-common/tests/test_vulnerabilities_controller.py]: Thank you

**Pull request merged**